api/src/wfl/api/spec.clj

```diff
@@ -32,9 +32,11 @@
 (s/def ::uuid-query (s/keys :opt-un [::uuid]))
 (s/def ::version string?)
 (s/def ::wdl string?)
+(s/def ::workflow_options map?)
 (s/def ::workload-request (s/keys :opt-un [::input
                                            ::items
-                                           ::common_inputs]
+                                           ::common_inputs
+                                           ::workflow_options]
```

Comment:
Here's an idea! (Not sure if it's any good)
You could combine these into a "common" map that could look like: 

```
(s/def ::common (s/keys :opt-un [::inputs ::options]))
```

(This would also just require one call to `deep-merge` later in the implementation, just that's an extra)

Reply:
I like that idea--there's a spec thing called `s/merge` that lets you define multiple map specs and merge them together so we could do the sort of abstraction you're talking about. I didn't use it here because the "common" stuff is optional on requests and required on response, and we wouldn't have a great way to represent that.

Reply:
Are you suggesting another layer of nesting for common inputs and workflow options in the actual API?

Reply:
Yeah sure - they're both "common" things. Maybe rename "workflow_options" to just "options".

Reply:
I like keeping workflow_options because it helps differentiate itself from inputs and it the name is the same that Cromwell uses to help reference that authoritative documentation. I don't think we should invent a new name and add a layer of indirection, I think keeping it reinforces that WFL is transparent about them and just passes them along.

I'm indifferent about nesting this stuff inside a "common" key, I can change it if you really think it is important. I wish we'd fleshed that out during design rather than back-tracking now.